src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,114 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int newDefaultCapacity = DEFAULT_CAPACITY;
+    private T[] elementData;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values, no need to assign it if it is a default value